src/lib/constants.ts

```diff
@@ -9,6 +9,7 @@ const constants = {
 	rootMountPoint,
 	databasePath:
 		checkString(process.env.DATABASE_PATH) || '/data/database.sqlite',
+	containerId: checkString(process.env.SUPERVISOR_CONTAINER_ID) || undefined,
```

Comment:
@CameronDiver @robertgzr this is the env variable we'll use to set the CID with `--env-cid` parameter in `balena run`.

I didn't use BALENA_ prefix, since we don't use it for other variables.

Reply:
As long as this is definable in the engine, that's fine by me.